### PR TITLE
Added a Microsoft Graph PowerShell tab in the article

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by msewaweru** [comment]
Fixes #16322

**Comment by msewaweru** [comment]
@CelesteDG , please review this.

**Comment by CelesteDG** [comment]
@msewaweru - Please see the above message and take a look as it may need fixing

**Comment by msewaweru** [comment]
> @msewaweru - Please see the above message and take a look as it may need fixing

@CelesteDG, the suggestion is resolved. Thanks.

File: concepts/auth-cloudsolutionprovider.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ The initial steps required here follow most of the same steps used to register a
 
 Finally grant your partner-managed app those configured permissions for all your customers. You can do this by adding the **servicePrincipal** that represents the app to the *Adminagents* group in your Partner tenant, using Azure AD powershell V2. You can download and install Azure AD PowerShell V2 from [here](https://www.powershellgallery.com/packages/AzureAD).  Follow these steps to find the *Adminagents* group, the **servicePrincipal** and add it to the group.
```
Comment (FaithOmbongi) [inline concepts/auth-cloudsolutionprovider.md:39]: Suggestion to add reference to Microsoft Graph PS 
```suggestion
Finally grant your partner-managed app those configured permissions for all your customers. You can do this by adding the **servicePrincipal** that represents the app to the *Adminagents* group in your Partner tenant, using [Azure AD powershell V2](https://www.powershellgallery.com/packages/AzureAD) or [Microsoft Graph PowerShell](/powershell/microsoftgraph/installation). Follow these steps to find the *Adminagents* group, the **servicePrincipal** and add it to the group.
```

**Pull request merged**